workflows/nanopore.nf

```diff
@@ -115,7 +115,6 @@ workflow NANOPORE {
     ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
     ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
     ch_multiqc_files = ch_multiqc_files.mix(NANOPLOT.out.txt.collect().ifEmpty([]))
-    ch_multiqc_files = ch_multiqc_files.mix(PORECHOP_ABI.out.log.collect().ifEmpty([]))
```

Comment:
what changed here?

Reply:
I removed the multiqc output for this module because multiqc doesn't read the log file from this module unfortunately

Reply:
should creating a multiqc module for porechop be on a non-urgent wish list/to-do list somewhere so we're keeping track of it? or is that info not really that useful to include in multiqc anyway?

Reply:
there is a multiqc module for the normal porechop, but I think something about how porechop_abi outputs the log is a tad different and I just didn't want to mess with it for now. it would be nice and is non-urgent, I can create an issue for it.